amoro-ams/src/main/java/org/apache/amoro/server/dashboard/MixedAndIcebergTableDescriptor.java

```diff
@@ -451,19 +462,132 @@ public List<PartitionBaseInfo> getTablePartitions(AmoroTable<?> amoroTable) {
         getTableFilesInternal(amoroTable, null, null);
     try {
```

Comment:
Can use try catch with resource to close the io automaticly
```suggestion
        try (CloseableIterable<PartitionFileBaseInfo> tableFiles 
                 = getTableFilesInternal(amoroTable, null, null)) {
      for (PartitionFileBaseInfo fileInfo : tableFiles) {
        refreshPartitionBasicInfo(fileInfo, partitionBaseInfoHashMap);
      }
    } catch (IOException e) {
      LOG.warn("Failed to close the manifest reader.", e);
    }
```